sdk/cosmosdb/cosmos/src/utils/tracing.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { createTracingClient } from "@azure/core-tracing";
+import { Constants } from "../common/constants";
+
+/**
+ * Creates a span using the global tracer.
+ *
+ * @param name - The name of the operation being performed.
+ * @param tracingOptions - The options for the underlying http request.
+ *
+ * @internal
+ */
+export const tracingClient = createTracingClient({
```

Comment:
Do you plan to add any traces to the cosmos runtime yet, or is this groundwork to do so in the future?

Reply:
yes